README.md

```diff
@@ -7,76 +7,49 @@ _Simple, scalable state management._
 [![npm version](https://badge.fury.io/js/mobx.svg)](https://badge.fury.io/js/mobx)
 [![OpenCollective](https://opencollective.com/mobx/backers/badge.svg)](docs/backers-sponsors.md#backers)
 [![OpenCollective](https://opencollective.com/mobx/sponsors/badge.svg)](docs/backers-sponsors.md#sponsors)
-
 [![Discuss on Github](https://img.shields.io/badge/discuss%20on-GitHub-orange)](https://github.com/mobxjs/mobx/discussions)
 [![View changelog](https://img.shields.io/badge/changelogs.xyz-Explore%20Changelog-brightgreen)](https://changelogs.xyz/mobx)
 
 ---
 
-> Documentation for older **unsupported** V4/V5 can be [found here](https://github.com/mobxjs/mobx/blob/mobx4and5/docs/README.md), but be sure to read about [current documentation first](https://mobx.js.org/about-this-documentation.html).
-
-MobX is made possible by the generosity of the sponsors below, and many other [individual backers](https://github.com/mobxjs/mobx/blob/main/docs/backers-sponsors.md#backers). Sponsoring directly impacts the longevity of this project.
+## Documentation
 
-**🥇 Gold sponsors (\$3000+ total contribution):** <br/>
-<a href="https://mendix.com/"><img src="https://mobx.js.org/assets/mendix-logo.png" align="center" width="100" title="Mendix" alt="Mendix" /></a>
```

Comment:
> I believe it is better to let readers know what the project is about before showing a long list of sponsors, so I've moved the introduction above the list of sponsors.

It's a controversial topic. I believe the author of a library should decide where to put the sponsors link. Open Source projects help companies make billions, however only few companies are willing to pay back. You may find this article interesting: https://www.prnewswire.com/news-releases/survey-finds-many-open-source-maintainers-are-stressed-out-and-underpaid-but-persist-so-they-can-make-a-positive-impact-301304925.html

I don't think the change is necessary.

Reply:
Thank you for the reply, I will reverse this change.